lib/bots/procbot.ts

```diff
@@ -113,6 +119,36 @@ export class ProcBot<T> {
         this.output(level, this._alertLevel, this.alertLevelStrings, message);
     }
 
+    // Process a configuration file from YAML into a nested object.
+    protected processConfiguration(configFile: string): ProcBotConfiguration | void {
+        const config: ProcBotConfiguration = yaml.safeLoad(configFile);
+
+        if (!config) {
+            return;
+        }
+
+        // Swap out known tags that become booleans.
+        const minimumVersion = ((config || {}).procbot || {}).minimum_version;
+        if (minimumVersion && process.env.npm_package_version) {
+            config.procbot.minimum_version = (process.env.npm_package_version < minimumVersion) ? false : true;
+        } else {
+            config.procbot.minimum_version = true;
+        }
```

Comment:
I find it pretty strange that minimum version change from a number to a boolean. If you immediately fail here, rather than delegating using the boolean result to `.getConfiguration` later on, I don't think you'd need to do this - it's always a number, and you either move forwards, or give up immediately.